src/Illuminate/Support/Manager.php

```diff
@@ -15,15 +15,6 @@ abstract class Manager
      */
     protected $container;
 
-    /**
-     * The container instance.
-     *
-     * @var \Illuminate\Contracts\Container\Container
-     *
-     * @deprecated Use the $container property instead.
-     */
-    protected $app;
```

Comment:
Note that the container property has been there since at least 6.x, so people can write package code that works on Laravel 6, 7 and 8 easily, without needing separate code for 6 and 7+.